applications/asset_tracker_v2/tests/cloud_codec_nrf_mocked_cjson/src/cJSON_to_mock.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
```

Comment:
Would be nice to include the reasoning behind creating this file.

Reply:
added a comment about that.